src/opensles/AudioStreamOpenSLES.h

```diff
@@ -55,9 +55,9 @@ class AudioStreamOpenSLES : public AudioStreamBuffered {
      *
      * @return state or a negative error.
      */
-    StreamState getState() override { return mState; }
+    StreamState getState() override { return mState; } // const?
 
-    int32_t getFramesPerBurst() override;
+    int32_t getFramesPerBurst() override;  // const?
```

Comment:
AudioStreamAAudio::getFramesPerBurst() makes some non-const assignments. I think this has to stay non-const.